packages/bitcoin/src/validation/transaction-validation.spec.ts

```diff
@@ -45,16 +60,38 @@ describe('isValidBitcoinTransaction', () => {
   it('should throw an error if the amount is insufficient', () => {
     const transaction = {
       ...mockTransaction,
-      amount: createMoney(0, 'USD'),
+      amount: createMoney(0, 'BTC'),
     };
     expect(() => isValidBitcoinTransaction(transaction)).toThrow(BitcoinError);
   });
 
   it('should throw an error if the balance is insufficient', () => {
     const transaction = {
       ...mockTransaction,
-      amount: createMoney(200, 'USD'),
+      amount: createMoney(200, 'BTC'),
     };
     expect(() => isValidBitcoinTransaction(transaction)).toThrow(BitcoinError);
   });
+
+  it('should throw an error if the amount precision is invalid', () => {
```

Comment:
@fbwoolf can you please help confirm this is correct for the tests?

I'm trying to test various amounts like `1.123456789 BTC` and `0.123456789 BTC` but I'm not sure if I'm using create money properly to create the amount for them

Reply:
`createMoney` won't shift the decimals but `createMoneyFromDecimals` will, so to throw an error you would use `createMoney(0.123456789, 'BTC')` bc that would be 9 decimals instead of 8. `createMoney` expects the fractional unit.

Reply:
Same error in the extension...

![Screenshot 2025-02-24 at 9 51 06 AM](https://github.com/user-attachments/assets/19bc597f-e61e-45e2-b0b5-d6273d131e61)

Reply:
If you ever use `createMoneyFromDecimals` it will shift the value 8 decimals (or the provided symbol decimals) expecting the base unit as the input.

Reply:
Thanks @fbwoolf ! I'm trying to tap into your wisdom as I go 🙇